src/main/xerus/monstercat/tabs/TabPlaylist.kt

```diff
@@ -0,0 +1,115 @@
+package xerus.monstercat.tabs
+
+import javafx.scene.control.ContextMenu
+import javafx.scene.control.Label
+import javafx.scene.control.SelectionMode
+import javafx.scene.control.TableRow
+import javafx.scene.control.TableView
+import javafx.scene.input.ClipboardContent
+import javafx.scene.input.DataFormat
+import javafx.scene.input.KeyCode
+import javafx.scene.input.MouseButton
+import javafx.scene.input.TransferMode
+import xerus.ktutil.javafx.MenuItem
+import xerus.ktutil.javafx.TableColumn
+import xerus.ktutil.javafx.fill
+import xerus.ktutil.javafx.properties.listen
+import xerus.monstercat.api.Player
+import xerus.monstercat.api.Playlist
+import xerus.monstercat.api.response.Track
+
+
+class TabPlaylist: VTab() {
+	private var table = TableView<Track>().apply {
```

Comment:
this table initalisation is quite long, which smells unclean, but I myself have no idea how to improve it

Reply:
Create new helper functions ? Or create a base class derivate of TableView to put it in a different file ?

Reply:
1) Only if they are reusable
2) I have thought of that, but then this file would be almost useless, and that extra class would not really serve a separate purpose.

Reply:
Both options aren't that great in that use case :confused: